src/main/java/it/eng/idsa/businesslogic/configuration/ApplicationConfiguration.java

```diff
@@ -42,6 +44,22 @@ public String getDapsUrl() {
 		return dapsUrl;
 	}
 
+	public int getWssSelfDescriptionPort() {
```

Comment:
Yes, we can!

And again - yes, the new port is required for fetching self-description in WSS flow but trying to avoid adding new property in application.properties, since it can either confuse the user or even them forgot to insert it, this was a safer way to proceed.

Reply:
I would put in both places (maybe not best situation, but better to cover both places)